src/main/daml/Daml/Finance/Claims/Util.daml

```diff
@@ -24,4 +24,3 @@ toTime forwardMap c =
     -- currently the contramap from `Time` back to `a` is not used, because `Observation`\s do not
     -- depend on time explicitly
   in mapParams contraMap forwardMap identity identity identity c
-
```

Comment:
Should leave empty line at the end of the file

Reply:
there were 2 empty lines, and I only removed 1

Reply:
seems like github doesn't display empty lines at the end